sds_data_manager/stacks/sds_data_manager_stack.py

```diff
@@ -44,6 +47,10 @@ def __init__(
         dynamodb_stack: DynamoDB,
         processing_step_function_arn: str,
         env: cdk.Environment,
+        db_secret_name: str,
+        vpc: ec2.Vpc,
+        vpc_subnets,
```

Comment:
I was wondering why we pass in both `vpc` and `vpc_subnets` but later saw that it was RDS's subnet selection. may be, we could rename this to something along there to indicate what these subnets are from and what it's used for? does RDS uses private subnet? I wonder if we can get private subnet from vpc and use that. I have not tried so it's ok. That's something we can look at later time.